nixos/modules/services/audio/squeezelite.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  uid = config.ids.uids.squeezelite;
+  cfg = config.services.squeezelite;
+
+in {
+
+  ###### interface
+
+  options = {
+
+    services.squeezelite= {
+
+      enable = mkOption {
```

Comment:
`mkEnableOption` is a short-hand for this. Note that it expands to something like "Whether to enable foo", so take that into account when choosing its argument.